travis/upload_packages.sh

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env bash
 set -ev
 
+# use python from pyenv
```

Comment:
Hmmm, why we need this duplication here ? 

https://github.com/cliqz-oss/keyvi/blob/master/travis/setup_pyenv.sh#L10
Maybe we just need to add export keyword here ?